fastlane/Fastfile

```diff
@@ -54,16 +54,31 @@ platform :ios do
   end
   
   desc "Upload a local IPA to Crashlytics with (group)."
+  desc "#### Options"
+  desc " * **`generate_changelog`**: Whether or not to generate a changelog as the release notes (boolean – default: `false`)"
+  desc " * **`changelog_source`**: The source to use to generate the changelog (e.g. `\"git\"`, `\"jenkins\"` – default: `"git"`)"
```

Comment:
I think we need some additional options here. For some projects, the git changes may pick up so many commits that the maximum changelong length is exceeded. For those, we've just been using the PR name, which is a safe default. You can take a look at the DTE Fastfile to see how this is generated.